Example/Podfile.lock

```diff
@@ -3,7 +3,7 @@ PODS:
     - InputBarAccessoryView/Core (= 4.3.2)
   - InputBarAccessoryView/Core (4.3.2)
   - MessageKit (3.1.0):
-    - InputBarAccessoryView (~> 4.3.2)
+    - InputBarAccessoryView (~> 4.3.0)
```

Comment:
We want this at 4.3.2. The swift package manager stuff is on `master` after the 4.3.2 tag.

Reply:
It is pointed to 4.3.2 thanks to `~>` which means it's the highest `4.3.x`

Reply:
As you can see above that btw. :)

Reply:
🤦‍♂